### PR TITLE
openstackclient: Add python-designateclient dep

### PR DESCRIPTION
**Pull request by ggustafsson**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Add missing dependency `python-designateclient` that is responsible for subcommands:

- openstack dns
- openstack ptr
- openstack recordset
- openstack tld
- openstack tsigkey
- openstack zone

https://pypi.org/project/python-designateclient/

### CODE REVIEW

File: Formula/o/openstackclient.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ class Openstackclient < Formula
   url "https://files.pythonhosted.org/packages/4b/4a/4fa8b9e681dbd38ecf2a86e05f0f9fa48eb4e31e7c4f5925877367022127/python-openstackclient-6.3.0.tar.gz"
   sha256 "9ba0bdf8d03087e585b40a8dcc429cebbdd5fdec24c1d60a102bf9f331320df1"
   license "Apache-2.0"
+  revision 1
```
Comment (SMillerDev) [inline Formula/o/openstackclient.rb:9]: Since this I the first mention I've seen of this I don't think a revision bump is needed.
```suggestion
```

Comment (ggustafsson) [reply Formula/o/openstackclient.rb:9]: Oh, I thought it was necessary when not bumping version. I’ll remove the line.

Comment (SMillerDev) [reply Formula/o/openstackclient.rb]: It is required to force an update for users, so we use it when that is needed. In this case I don't think it is though.

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
Thanks @ggustafsson!

**Pull request merged**